dbms/src/Columns/ColumnString.cpp

```diff
@@ -275,6 +275,50 @@ ColumnPtr ColumnString::replicate(const Offsets & replicate_offsets) const
     return res;
 }
 
+ColumnPtr ColumnString::replicate(size_t start_row, size_t end_row, size_t prev_offset, const IColumn::Offsets & replicate_offsets) const
+{
+    size_t col_size = size();
```

Comment:
The variable name is hard to understand, is it `rows`?

Reply:
rename.